source/vistas/core/graphics/vector/geometry.py

```diff
@@ -0,0 +1,72 @@
+import numpy
+from OpenGL.GL import *
+
+from vistas.core.graphics.geometry import InstancedGeometry
+
+
+class VectorGeometry(InstancedGeometry):
```

Comment:
This feels plugin-specific. Is rendering an arrow based on raster inputs generic enough to justify a place in core?

Reply:
So it's not necessarily based on raster inputs alone. The specification for using a `VectoryGeometry` is flexible enough that as long as there is enough data specified (i.e. 7 floats for each vector), then it doesn't really matter where the data comes from. While it is only used in one visualization currently, I don't think that it is unlikely that it could be used elsewhere. For example, a VectorGeometry could be used to dictate a line of vectors for a wind direction, or be used in something like Larry Mahrt's visualization, where each of the vectors will have their direction altered by some polar degree for each timestep.

Reply:
Ok, sounds good.